tests/console/krb5.pm

```diff
@@ -0,0 +1,125 @@
+# SUSE's openQA tests
+#
+# Copyright 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: MIT kerberus 5 test (krb5)
+# Maintainer: Sergio Rafael Lemke <slemke@suse.cz>
+
+# Setup a kdc (kdc is the Kerberos version 5 Authentication Service and Key Distribution Center)
+# and run following commands: addprinc, getprincs, modprinc, getprinc, delprinc, listpols,
+# addpol, modpol, delpol, getprivs, ktremove.
+# system units tests (restart, stop, start, staus);
+# rckadmind service start, stop, restart, status
+
+use base 'consoletest';
+use utils qw(zypper_call systemctl);
+use strict;
+use warnings;
+use testapi;
+
+sub run {
+    select_console 'root-console';
+    zypper_call 'in krb5 krb5-server krb5-client';
+
+    #Get the script that creates the kerberus server:
+    script_run 'wget ' . data_url('console/setup-kerberos-server-3.pl');
+    script_run 'chmod +x setup-kerberos-server-3.pl';
+
+    #backup VM current status/setup:
+    script_run 'cp /etc/hosts /etc/hosts-bkp';
+    script_run 'cp /etc/hostname /etc/hostname-bkp';
+    script_run 'hostname > /tmp/hostname';
+
+    #log original hostname:
+    record_info 'Current Hostname';
+    assert_script_run 'hostname';
+
+    #setup FQDN hostname to successfully create krb5 server:
+    script_run 'hostname openqa.kerberus.org';
+    script_run 'echo "127.0.0.1 openqa.kerberus.org openqa" > /etc/hosts';
+    script_run 'echo "openqa.kerberus.org" > /etc/hostname';
+
+    #confirm FQDN:
+    assert_script_run 'hostname -f';
+
+    #Test Case 1439086: MIT Kerberos server:
+    record_info 'Testcase 1439086:';
+    assert_script_run './setup-kerberos-server-3.pl --realm openqa.kerberus.org -noy2kc --noinstall';
+
+    #Fix realm address on krb5.cnf file:
+    assert_script_run "sed -i 's/openqa.kerberus.org.kerberus.org/openqa.kerberus.org/g' /etc/krb5.conf";
+
+    #Test Case 1439087: ssh with MIT Kerberos 5:
+    record_info 'Testcase 1439087:';
+
+    #kadmind starts the Kerberos administration server systemd unit:
+    systemctl('restart kadmind', expect_true => 1);
```

Comment:
Where did you find `expect_true` ? Can't find it anywhere documented.

Reply:
hmm, good question! 
Well there is "expect_false" all around, maybe "expect_true" is just redundant, will check on it.

Reply:
As far as I know 'expect_true' is the default